content/en/docs/reference/command-line-tools-reference/feature-gates/AnyVolumeDataSource.md

```diff
@@ -12,7 +12,10 @@ stages:
     toVersion: "1.23"
   - stage: beta
     defaultValue: true
-    fromVersion: "1.24"
+    fromVersion: "1.32"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.33"
```

Comment:
Please try add `locked: true/false` to indicate whether this gate is "lockedToDefault".
It is not used as of today, but we may need to expose this to end users some day.